Classes/NodeProxyGui2.sc

```diff
@@ -169,6 +169,14 @@ NodeProxyGui2 {
 			paramViews[key][\numBox].value_(spec.constrain(val));
 			paramViews[key][\slider].value_(spec.unmap(val));
 		}
+		{ val.isArray && {val.every(_.isNumber)} } {
+			if(paramViews[key][\type] != \array, { this.makeParameterSection() });
+			spec = params[key].value;
+			val.do { |v, i|
+				paramViews[key][\numBoxes].wrapAt(i).value = spec.wrapAt(i).constrain(v);
+				paramViews[key][\sliders].wrapAt(i).value = spec.wrapAt(i).unmap(v)
+			};
+		}
 		{ val.isArray } {
 			if(paramViews[key][\type] != \array, { this.makeParameterSection() });
 			spec = params[key].value;
```

Comment:
a bit of code duplication here.  i believe the val.every(_.isNumber) test above can be done here using an if statement.  i mean like this...
```
{ val.isArray } {
	if(paramViews[key][\type] != \array, { this.makeParameterSection() });
	spec = params[key].value;
	if(val.every(_.isNumber), {
		val.do...
		...
	}, {
		paramViews[key]...
		...
```